package.json

```diff
@@ -61,6 +61,8 @@
     "eslint-plugin-compat": "^3.5.1",
     "eslint-plugin-escompat": "^1.1.0",
     "eslint-plugin-github": "^3.4.1",
+    "jsdom": "16.3.0",
+    "jsdom-global": "3.0.2",
```

Comment:
I added `jsdom` as a quick way to write a test for this but I would probably want to replace it with karma in order to be more in line with our other repositories.